extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -180,6 +179,48 @@ private function prepareWikiVariables() {
 		return $wikiVariables;
 	}
 
+	public function getMobileWikiVariables() {
+		( new CrossOriginResourceSharingHeaderHelper() )->allowWhitelistedOrigins()
+			->setAllowMethod( [ 'GET' ] )
+			->setHeaders( $this->response );
```

Comment:
is it needed here? If it's internal traffic only I think we can remove it

Reply:
@mszabo-wikia requests to these endpoints come also from front-end apps

Reply:
@rybmat which ones? I thought all Ember clients use a server-side request.